index.src.html

```diff
@@ -352,32 +344,18 @@ <h3 id="header">
   [[!RFC5234]]:
 
   <pre class="abnf" link-type="grammar" dfn-type="grammar">
-    Report-To = <a>json-field-value</a>
-                ; See Section 2 of [[HTTP-JFV]], and Section 2 of [[RFC7159]]
+    Clear-Site-Data = "Clear-Site-Data" ":" <a>clear-site-data-value</a>
+    <dfn>clear-site-data-value</dfn> = OWS 1#(<a>type</a> OWS)
+    <dfn>type</dfn> = atom
   </pre>
 
-  The header's value is interpreted as an array of JSON objects, as described in
-  Section 4 of [[HTTP-JFV]].
-
-  Each object in the array represents a clearing action that the user agent MUST
-  undertake, and will be parsed as defined in [[#parsing]].
-
-  The following subsections defined the initial set of known members in each
-  JSON object the header's value defines. Future versions of this document may
-  define additional such members, and user agents MUST ignore unknown members
-  when parsing the header.
-
-  <h4 id="types-member">
-    The `types` member
-  </h4>
+  The header's value is interpreted as an array of atoms.
 
-  The <dfn>`types`</dfn> member is an array of keywords designating the kinds
-  of data that the server wishes the user agent to remove. The member's value
-  MUST be an array, and that array MUST contain only strings; any other types
-  will result in a parse error.
+  Each atom in the array represents a data type that the user agent MUST
+  clear, and will be parsed as defined in [[#parsing]].
 
-  The following are the initial set of known types which may be specified in
-  the member's array value. Future versions of this document may define
+  The following are the initial set of known types which may be specified as
+  the array's elements. Future versions of this document may define
   additional types, and user agents MUST ignore unknown types when parsing the
   header:
```

Comment:
Maybe something like `The <a grammar>data-type-value</a> grammar defines an initial set of known data types which can be cleared using this API. Future versions ...`, and then following it up with the list.